src/parseOption.ts

```diff
@@ -55,11 +56,21 @@ export function prepareCustomFieldsData(itemData: ScenegraphData, options: Optio
   const tooltipData: TooltipData[] = [];
 
   options.fields.forEach(function (fieldOption) {
+    const fieldStr = !isFunction(fieldOption.field) ? fieldOption.field : undefined;
+    const fieldFn =  isFunction(fieldOption.field)  ? fieldOption.field : undefined;
+    const titleStr = !isFunction(fieldOption.title) ? fieldOption.title : undefined;
+    const titleFn =  isFunction(fieldOption.title)  ? fieldOption.title : undefined;
+
     // prepare field title
-    const title = fieldOption.title ? fieldOption.title : fieldOption.field;
+    const title =
+      (titleFn && titleFn(itemData)) ||
+      titleStr ||
+      fieldStr;
 
     // get (raw) field value
-    const value = getValue(itemData, fieldOption.field, options.isComposition);
+    const value =
+      (fieldFn && fieldFn(itemData)) ||
```

Comment:
Since `fieldFn` is used to access and return value, not the field name, would it be clearer if it should be put in a separate key, like `valueAccessor`, not `field`.

Reply:
Oof, you're right! I'll refactor accordingly.

Reply:
Oof, you're right -- will refactor.